### PR TITLE
[Frontend] update counter message

### PR DESCRIPTION
**Pull request by Xiawpohr**
## Summary

Update message about how many actions user does in an epoch, which may affect anonymity. 

## Linked Issue

#398 

## Todo

-   [x] Refactor EpochInfo component
-   [x] Implement a function which get proper message based on action count

### CODE REVIEW

File: packages/frontend/src/features/core/components/EpochInfo/EpochInfo.tsx [hunk 1]
```diff
@@ -3,69 +3,96 @@ import Countdown from 'react-countdown'
...
+                <div
+                    key={index}
+                    className={clsx('flex-1', getCounterColor(index, count))}
+                />
+            ))}
+        </div>
+    )
+}
+
+function getAcountCountMessage(count: number) {
```
Comment (kittybest) [inline packages/frontend/src/features/core/components/EpochInfo/EpochInfo.tsx:61]: nit: typo: *getAccountCountMessage*
(少一個c）

**Review by kittybest — COMMENTED** [review]
Thanks! just a typo

File: packages/frontend/src/features/core/components/EpochInfo/EpochInfo.tsx (hunk 1 above)
Comment (Xiawpohr) [reply packages/frontend/src/features/core/components/EpochInfo/EpochInfo.tsx]: Fixed

**Review by kittybest — APPROVED** [review]
Thanks! just need to rebase

**Pull request merged**